packages/core/core-flows/src/order/workflows/update-order-changes.ts

```diff
@@ -0,0 +1,20 @@
+import { OrderChangeDTO, UpdateOrderChangeActionDTO } from "@medusajs/types"
+import {
+  WorkflowData,
+  WorkflowResponse,
+  createWorkflow,
+} from "@medusajs/workflows-sdk"
+import { updateOrderChangesStep } from "../steps"
+
+export const updateOrderChangesWorkflowId = "update-order-change"
+/**
+ * This workflow updates one or more order changes.
+ */
+export const updateOrderChangesWorkflow = createWorkflow(
```

Comment:
q: is this used anywhere?

Reply:
not really, I added it because I'm used to do it haha.
I will keep it anyway